### ISSUE TITLE
VM Tasks are not showing correct progress from shared directory

### ISSUE THREAD

**Issue by JacobWKlein** [BOINC/boinc#1368]
I'm not 100% if this is a bug or not, but based on discussion with Rom, it seems that it might be.

Basically, I have 2 RNA World tasks, that show the following progress %'s in BOINC 7.6.3 x64, on Windows 10 x64 Build 10166:
84.789%
6.793%

Yet, when I look at the slots directories, the vbox_job.xml files say:
<soft_link>../../projects/www.rnaworld.de_rnaworld/cms_vbox_job_1.12.xml</soft_link>

When I go to the projects directory to look at that "cms_vbox_job_1.12.xml" file, it says the following (Note: I replaced the '<' and '>' with '[' and ']' to get it to show semi-properly, here)

[vbox_job]
  [os_name]Debian_64[/os_name]
  [memory_size_mb]3072[/memory_size_mb]
  [enable_shared_directory/]
  [enable_remotedesktop/]
  [fraction_done_filename]shared/progress.txt[/fraction_done_filename]
  [minimum_checkpoint_interval]1800[/minimum_checkpoint_interval]
[/vbox_job]

When I go back to the slots directories, and look at the shared folders of them, their progress.txt files have the following contents (and they ARE being updated by the VM every half minute or so):
0.98765
0.039375

So, what is causing this discrepancy? Shouldn't the UI be showing the values from the progress.txt files??

Regards,
Jacob Klein

**Comment by JacobWKlein** [comment]
I did a lot of offline testing, using anonymous platform, and I figured out the problem here.

The Progress % is not being reported correctly, and the Time Remaining (Estimated) may be showing a malformed value. These issues are happening because the "cms_vbox_job*.xml" files have a <fraction_done_filename> value of "shared/progress.txt". Per my offline testing, this needs to be changed to only be set to "progress.txt" ... and the "shared/" text needs to be removed from that xml file... and then things will work correctly. NOTE: A user should NOT try to perform these changes on their data folder, because it'll cause their tasks to abort.

Rom mentioned that he normalized this setting to remove the need for the "shared/" text, at some point. I guess he did it in a VBoxWrapper version earlier than the 2-most-recent RNA VM app versions (VBoxWrapper versions 26167, 26179), because they are both busted.

Do we want any backwards compatibility support for vbox job xml files that happen to have the "shared/" text, so they can still work correctly without encountering problems? If not, then this issue can be closed out, due to "bad project configuration".

Thanks!
Jacob Klein
